docs/substrate-based-crab-chain/crab-tut-create-account.md

```diff
@@ -5,102 +5,38 @@ sidebar_label: Create an Account
 sidebar_position: 4
 ---
 
-There are mainly three ways to generate a Crab account, you can choose either one based on your preference.  Once your account is created and you have your account `secret phrase` or `secret seed`, you can migrate your account from various medium by importing your account.  
+If you are a user who has never used polkadot{.js} extension, please refer to the tutorial in the first part of this article; if you are a user who has used the old version of Darwinia Apps, please refer to the second part of this article to migrate the old account. For technically-advanced users who are comfortable with the command line and compiling Rust code, you can use the subkey to create your account, please refer to the third part of this article, and it is not recommended for general users.
 
-## 1. Crab Web Wallet
+> NOTE: Not all old accounts need to be migrated, and accounts that have been imported into the polkadot{.js} extension do not need to be migrated, meanwhile, these accounts will not be displayed in the `Account Migration`.
+> 
 
-![setting](../assets/substrate-based-crab-chain/create-account-01.png)
+## 1. How to Connect Polkadot{.js} Extension to Darwinia/Crab
 
-Enter [Crab Web Wallet](https://apps.darwinia.network), you can see two buttons "Add Account" and "Restore JSON" in the "Account" column.
+This part will walk you through the process of connecting your [polkadot{.js} extension](https://polkadot.js.org/extension/) to Darwinia.
 
-![create account](../assets/substrate-based-crab-chain/create-account-02.png)
+### Install the Polkadot{.js} Extension
 
-**New account**
+If you do not already have the polkadot{.js} extension installed, you can install the extension from the [Chrome store](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd). For help with installation, you can follow the Getting Started with the polkadot{.js} extension guide from the official polkadot{.js} extension documentation. Once you have installed the polkadot{.js} extension, follow the account creation wizard. Make sure to store your mnemonic safely and do not share it with anyone. If you prefer, there is also a video tutorial that you can check out [here.](https://www.youtube.com/watch?v=sy7lvAqyzkY)
 
-Click "Add Account", after setting the basic account information, click the "Save" button. (By default, only "mnemonic", "private key" need to be switched)
+When the polkadot{.js} extension is fully set up, you’ll have your address displayed at the top of the extension window.
 
-![create account](../assets/substrate-based-crab-chain/create-account-03.png)
+### Connect to Darwinia Apps
```

Comment:
How about changing this title to `Migrate From Legacy Darwinia Apps`? The latest Darwinia Apps does not provide create account functionality in my view. Just for account migration.